### ISSUE TITLE
Release detokenizer training code

### ISSUE THREAD

**Issue by hzphzp** [AILab-CVC/SEED-X#11]
Hi,  
  
I'm particularly interested in the training code for your two-phase training process for the detokenizer.

Is it possible to open source the alignment and fine-tuning code for ViT and SDXL Unet models? Access to this code would be incredibly helpful for my research and others in the field.  
  
Thanks for considering this request.  
  
Best.

**Comment by geyuying** [comment]
Hi, we are not going to release the visual de-tokenization training code at present 😁.
